lisp/ess-custom.el

```diff
@@ -2050,7 +2050,17 @@ See also function `ess-create-object-name-db'.")
     "switch" "function" "return" "on.exit" "stop" ".Defunct" "tryCatch"
     "withRestarts" "invokeRestart"
     "recover" "browser")
-  "Reserved words or special functions in the R language.")
+  "Reserved words or special functions in the R language.
+See also `ess-R-keystrings'.")
+
+(defvar ess-R-keystrings
```

Comment:
the goal here was to remain consistent with the naming of `ess-R-keywords', since `keystrings' is somehow it's complement; if you confirm that would you prefer to hide the `keystrings' variable from the public, I will rename it as you suggested

Reply:
I've implemented the suggested changes; `ess-R-keystrings` is now `ess-r--keystrings` and I removed it from the docstring of `ess-R-keywords`